### PR TITLE
Index data into embedded Elasticsearch index

### LINKED ISSUE #1: Index data into embedded Elasticsearch index

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #1

### CODE REVIEW

**Comment by acka47** [comment]
Looks good for a first version. I will look at the details after my holidays. One thing to be corrected: Creators and their website's URL are listed in the `creator` array of a collection, e.g.:

```json
"creator": [
  "Berlin-Brandenburgische Akademie der Wissenschaften",
  "http://bibliothek.bbaw.de"
]
```

This should rather be an object:

```json
"creator": {
  "name": "Berlin-Brandenburgische Akademie der Wissenschaften",
  "url": "http://bibliothek.bbaw.de"
}
```

We might even use lobid-organisations URIs as id, e.g.:

```json
"creator": {
  "id": "http://beta.lobid.org/organisations/DE-B4#!"
  "name": "Berlin-Brandenburgische Akademie der Wissenschaften",
  "url": "http://bibliothek.bbaw.de"
}
```

File: README.textile [hunk 1]
```diff
@@ -27,3 +27,39 @@ Get @digitalisiertedrucke@, set up the Play application, and run the tests:
 * @cd git/digitalisiertedrucke ; ~/activator-1.3.10-minimal/bin/activator test@
 
 See the @.travis.yml@ file for details on the CI config used by Travis.
+
+h1. Data import
+
+To transform the bzipped Marc-XML source data to JSON and index into an embedded Elasticsearch index, run:
+
+@~/activator-1.3.10-minimal/bin/activator "runMain ImportData"@
```
Comment (SBRitter) [inline README.textile]: How do I know when this is finished? Log is simply showing nothing new, right?
(Maybe add info to open a new shell or cancel running process for the next step )

File: conf/application.conf [hunk 2]
```diff
@@ -15,6 +15,16 @@
 # HOCON will fall back to substituting environment variable:
 #mykey = ${JAVA_HOME}
 
+## Digitalisiertedrucke
+
+index {
+	host = "localhost"
+	name = "digitalisiertedrucke"
+	location = "."
+	http_port = "6011"
```
Comment (SBRitter) [inline conf/application.conf]: In organisations, we have them named like this: "index.es.port.http" -- with full stops. Do we have naming conventions for this? Do we care?

**Comment by SBRitter** [comment]
Travis checks fail after commits by @acka47. Assigning him.

File: README.textile (hunk 1 above)
Comment (fsteeg) [inline README.textile]: This is not like in lobid-organisations, where we transform and index when the app starts, and the process keeps running after indexing is done. Here, we just run that main class, so it's done when we process finishes and we get back the command line prompt. It also logs to the console for me (both our own app info and some Elasticsearch logging).

File: conf/application.conf (hunk 2 above)
Comment (fsteeg) [inline conf/application.conf]: The notation with dots is equivalent to the nesting, i.e. the nested properties here can be accesses with `index.host` etc. In organisations, it's actually very deeply nested: `index.es.port.http` is actually 4 levels deep. That would not look very clean if written nested. On the other hand, if it's only one level like here, I think its very readable and provides nice grouping. So I'd say it depends on the concrete use case which notation makes more sense.

**Comment by SBRitter** [comment]
+1

**Pull request merged**